docs/operation/changing_control_plane_ingress_ip.rst

```diff
@@ -18,6 +26,15 @@ Changing the Control Plane Ingress IP
       $ salt-call metalk8s_network.get_control_plane_ingress_ip
       local:
           <my-new-ip>
+      $ salt-call pillar.get networks:control_plane
+      local:
+        ----------
+        cidr:
+          - <control-plane-cidr>
+        ingress:
+          ip:
+            <my-new-ip>
+        metalLB: <my-new-metallb-config>
```

Comment:
Shouldn't this reflect the same as what we set in BootstrapConfiguration?

```suggestion
        metalLB:
          enabled: <true | false>
```

Reply:
Right I forgot to update this one (basically in older version of this PR MetalLB config was there also) I will update it